src/Components/Postulants/postulants.module.css

```diff
@@ -1,6 +1,47 @@
 .container {
-  flex: 1;
   display: flex;
+  flex-direction: column;
   justify-content: center;
   align-items: center;
+}
+
+a {
```

Comment:
avoid using tag name selectors

Reply:
added classes to avoid most tag name selectors